packages/ccip-js/package.json

```diff
@@ -12,12 +12,13 @@
     "check": "tsc --noEmit",
     "build:watch": "tsc -w",
     "build": "tsc && hardhat compile",
-    "lint": "eslint 'src/**/*.{ts,js}'",
+    "lint": "eslint 'src/**/*.{ts,js}' 'test/**/*.{ts,js}'",
     "format": "prettier --write 'src/**/*.{ts,js,json,md}'",
-    "t:int": "jest --coverage -u --testMatch=\"**/integration-testnet**.test.ts\" --detectOpenHandles",
-    "t:int:viem": "jest --coverage -u --testMatch=\"**/integration-testnet.test.ts\" --detectOpenHandles",
-    "t:int:ethers": "jest --coverage -u --testMatch=\"**/integration-testnet-ethers.test.ts\" --detectOpenHandles",
-    "t:unit": "jest --coverage -u --testMatch=\"**/unit.test.ts\" ",
+    "t:int": "jest --coverage -u --testMatch=\"**/integration-testnet**.test.ts\" --runInBand --detectOpenHandles --forceExit",
+    "t:int:viem": "jest --coverage -u --testMatch=\"**/integration-testnet.test.ts\" --runInBand --detectOpenHandles --forceExit",
```

Comment:
--forceExit is generally not recommended.  If its the best way for now then perhaps update the comment I left in integration test file, line 22 or so that is marked with a TODO? just to explain the context?